src/main/java/uk/gov/hmcts/divorce/bulkaction/ccd/event/SystemEmptyCase.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.divorce.bulkaction.ccd.event;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.sdk.api.CCDConfig;
+import uk.gov.hmcts.ccd.sdk.api.ConfigBuilder;
+import uk.gov.hmcts.divorce.bulkaction.ccd.BulkActionPageBuilder;
+import uk.gov.hmcts.divorce.bulkaction.ccd.BulkActionState;
+import uk.gov.hmcts.divorce.bulkaction.data.BulkActionCaseData;
+import uk.gov.hmcts.divorce.divorcecase.model.UserRole;
+
+import static uk.gov.hmcts.divorce.bulkaction.ccd.BulkActionState.Created;
+import static uk.gov.hmcts.divorce.bulkaction.ccd.BulkActionState.Empty;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.SYSTEMUPDATE;
+import static uk.gov.hmcts.divorce.divorcecase.model.access.Permissions.CREATE_READ_UPDATE;
+
+@Component
+public class SystemEmptyCase implements CCDConfig<BulkActionCaseData, BulkActionState, UserRole> {
+
+    public static final String SYSTEM_EMPTY_CASE = "system-empty-case";
+
+    @Override
+    public void configure(final ConfigBuilder<BulkActionCaseData, BulkActionState, UserRole> configBuilder) {
+        new BulkActionPageBuilder(configBuilder
+            .event(SYSTEM_EMPTY_CASE)
+            .forStateTransition(Created, Empty)
+            .name("Set empty case")
+            .description("Set empty case")
```

Comment:
Minor thing really but perhaps we could have a slightly more descriptive description for anyone who doesn't really know what the empty case functionality is for or what it does?

Reply:
Might not be necessary though since it is merely a state transition event